prometheus_client/multiprocess.py

```diff
@@ -4,6 +4,11 @@
 
 from collections import defaultdict
 
+try:
+    from collections import OrderedDict
+except ImportError:
+    OrderedDict = dict
```

Comment:
The code should still work on 2.6

Reply:
The code works, but the order is not preserved on 2.6, so it's different behaviour

Reply:
Rather than changing this, how about changing the format of the multiprocess key to be a dict for the labels and using sort_keys=True?